source/hdev/Camera/Basic/CaptureHDRLoop.hdev

```diff
@@ -30,6 +30,8 @@
 <c></c>
 <c>* Connecting to Zivid camera</c>
 <l>open_framegrabber ('GenICamTL', 1, 1, 0, 0, 0, 0, 'progressive', -1, 'default', -1, 'false', 'default', Device, 0, 0, AcqHandle)</l>
+<l>get_framegrabber_param (AcqHandle, 'DeviceModelName', Value_DeviceModelName)</l>
```

Comment:
I think we agreed to use ThisFormForVariableNames for Halcon code, which is apparently standard for Haclon. YI, this_form_for_procedures is also apparently standard for Halcon for procedures.

Reply:
I would also skip 'Value' and just call it 'DeviceModelName'

Reply:
Done